Signal/src/ViewControllers/ConversationView/ConversationInputToolbar.m

```diff
@@ -73,15 +96,8 @@ - (void)createContents
     [self.attachmentButton addTarget:self
                               action:@selector(attachmentButtonPressed)
                     forControlEvents:UIControlEventTouchUpInside];
-
-    //    [_attachButton setFrame:CGRectMake(0,
-    //                                       0,
-    //                                       JSQ_TOOLBAR_ICON_WIDTH + JSQ_IMAGE_INSET * 2,
-    //                                       JSQ_TOOLBAR_ICON_HEIGHT + JSQ_IMAGE_INSET * 2)];
-    //    _attachButton.imageEdgeInsets
-    //    = UIEdgeInsetsMake(JSQ_IMAGE_INSET, JSQ_IMAGE_INSET, JSQ_IMAGE_INSET, JSQ_IMAGE_INSET);
     [self.attachmentButton setImage:[UIImage imageNamed:@"btnAttachments--blue"] forState:UIControlStateNormal];
-    [self addSubview:self.attachmentButton];
+    [self.leftButtonWrapper addSubview:self.attachmentButton];
```

Comment:
My feeling is that "attachment" should always be on the left and "send/voice memo" should always be on the right. Here's my argument.

* Regardless of RTL/LTR, most people are right-handed.
* We need to support one-handed use (e.g. while holding a subway pole).
* In one-handed mode, you use your thumb.
* "Send" is the key action; it needs to be highly accessible by the thumb.

Reply:
WDYT?

Reply:
iMessage keeps “Attachment” on the left and “Voice Memo” on the right for both LTR and RTL languages. In fact, even the “Send” arrow appears on the right for both language directions.

<img width="375" alt="messages-rtl" src="https://user-images.githubusercontent.com/2874864/31695936-7f1fa80a-b362-11e7-82d2-69dd70cb79aa.png">

Reply:
Thanks @erikstrottmann